docs/source/hyper-parameter-search.rst

```diff
@@ -185,34 +238,33 @@ expensive early steps, this can be a big win when performing a grid search.
 Incremental Hyperparameter Optimization
 ---------------------------------------
 
-The second category of hyperparameter optimization uses *incremental*
-hyperparameter optimization. These should be used when your full dataset doesn't
-fit in memory on a single machine.
-
 .. autosummary::
    dask_ml.model_selection.IncrementalSearchCV
 
-Broadly speaking, incremental optimization starts with a batch of models (underlying
-estimators and hyperparameter combinations) and repeatedly calls the underlying estimator's
-``partial_fit`` method with batches of data.
-
 .. note::
 
    These estimators require the optional ``distributed`` library.
 
-Here's an example training on a "large" dataset (a Dask array) with the
-``IncrementalSearchCV``.
+These are make repeated calls to the ``partial_fit`` method of the estimator.
+Naturally, these classes determine when to stop calling ``partial_fit`` by
+`adapting to previous calls`. The most basic level of this is to stop training
```

Comment:
asterisks.

Reply:
This reads a bit awkward. Maybe:

IncrementalSearchCV implements the most basic adaptation scheme: training stops when the score stops improving.